contracts/moda-contracts/src/interfaces/IModaRegistry.sol

```diff
@@ -2,20 +2,39 @@
 pragma solidity 0.8.21;
 
 interface IModaRegistry {
+    event CatalogMembershipSet(string name, address indexed membership);
+    event CatalogRegistered(string name, address indexed catalog);
```

Comment:
We should index the name too. indexing makes it easier to filter by the name you are looking for, which is more likely than someone searching by address. We should index both